### DISCUSSION TITLE
Truncated content at ToolTip when italic font style is applied

### DISCUSSION THREAD

**Discussion by dubrovenskaya** [AvaloniaUI/Avalonia#8737] [Q&A] [answered]
![ToolTip](https://user-images.githubusercontent.com/33326526/184387576-ff6c6318-5665-421c-aeab-2d4b5b11c25c.png)
Hi all, does anyone have any ideas on how I can avoid text truncation? This happens when italic font style is applied. From DevTool I can't determine which subelement needs to be styled, e.g. with padding. I thought it could be done this way:
![image](https://user-images.githubusercontent.com/33326526/184389239-44a1d08e-c169-4ef4-8ef9-898d94fa81d2.png)

Thank you for your reply!

**Comment by timunie** [comment]
You can try to use this Style-Selector for examle:

https://github.com/AvaloniaUI/Avalonia/blob/stable/0.10.x/src/Avalonia.Themes.Fluent/Controls/ToolTip.xaml#L76-L78

**Reply by dubrovenskaya** [reply]
![image](https://user-images.githubusercontent.com/33326526/184551911-452c8dbd-0a04-4b9b-96ca-a74058904ef1.png)

The problem is actually not that the text is cut off, but that the last character is not completely visible e.g. if it is "t" or "d". So that the last stroke is not visible as you can see in this image also. The problem exists only when using italic font.

**Comment by dubrovenskaya** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/33326526/184552320-e3c7f0bf-5ef3-47a5-8e86-3b5394ba847a.png)
![image](https://user-images.githubusercontent.com/33326526/184552364-0502ea52-6067-4054-825f-bb6fd07e4930.png)

The Problem is solved using Padding.
Thank you for the answer.

**Reply by timunie** [reply]
I think rhis issue will also be solved in next major release. @Gillibald did a lot of work to improve text rendering.
